pkg/crc/constants/constants.go

```diff
@@ -111,6 +112,19 @@ func GetDefaultBundle(preset crcpreset.Preset) string {
 	return bundles[runtime.GOOS]
 }
 
+func GetBundleNameFromBundlePath(preset crcpreset.Preset, bundlePath string) string {
```

Comment:
To be very generic this function dealing with `BundlePath` string which can be URI or local path and return the `BundleName` May be I will put `GetBundleNameFromBundlePathString` to make it more clear? Otherwise every place we have to first check if it use URI or path and then have to condition at every place to get the bundleName.

Reply:
I'm only suggesting to replace 'path' by 'uri' in this function as it feels a bit more representative of the kind of data it handles.